src/components/Provider.js

```diff
@@ -43,6 +47,7 @@ if (process.env.NODE_ENV !== 'production') {
     }),
     context: PropTypes.object,
     children: PropTypes.any,
+    ssr: PropTypes.bool,
```

Comment:
This isn't needed, right?

Reply:
Good catch.

Reply:
I've decided to go back to the `ssr` prop so this is need now.